scripts/build_docker_containers

```diff
@@ -21,17 +21,25 @@ build_mil_docker_image()
   docker build $DOCKER_ARGS $DOCKER_BASE_PATH/$1 -t $MIL_DOCKER_TAG_ROOT:$1 --build-arg MIL_DOCKER_TAG_ROOT=${MIL_DOCKER_TAG_ROOT}
 }
 
-build_vrx_images()
+build_vrx_image()
 {
- #build nvidia image
-#  .$(realpath $(dirname $BASH_SOURCE)/../)Navigator/simulation/vrx/docker/build.bash -n $(realpath $(dirname $BASH_SOURCE)/../)Navigator/simulation/vrx/
- 
- #build regular image
-  $(realpath $(dirname $BASH_SOURCE)/../)/NaviGator/simulation/vrx/docker/build.bash $(realpath $(dirname $BASH_SOURCE)/../)/NaviGator/simulation/vrx/
+  # see if user is nvidia gpuer
+  local NVIDIA="`grep -c \"X Driver\"  /var/log/Xorg.0.log`"
```

Comment:
This file is not present on my system. Have you tested this?

Reply:
Yes. On your system it should build the non Nvidia image. See build kite build on zobelisk to see it builds Nvidia image on Nvidia systems.

Reply:
Confirmed produces two error on my system. This does not seem to be a reliable way to check the existence of nvidia, may be helpful to see how OSRF does it

```
grep: /var/log/Xorg.0.log: No such file or directory
./scripts/build_docker_containers: line 29: [: ==: unary operator expected
```

Reply:
command is run on silent mode so that if command fails, the non-nvidia version is installed. OSRF does not automatically detect if your computer has NVIDIA GPU or not.